### PR TITLE
Update functions tables docs

### PR DESCRIPTION
**Pull request by car12o**
##### Summary

Update functions tables docs

### CODE REVIEW

**Comment by Ancairon** [comment]
@car12o this file is not on Learn currently, I get this is a docs update and I'm reviewer, do you want to add it in Learn?

**Comment by car12o** [comment]
> @car12o this file is not on Learn currently, I get this is a docs update and I'm reviewer, do you want to add it in Learn?

I don't think we should add this to learn for now.
Functionality is not yet available for users.

File: src/collectors/plugins.d/functions-table.md [hunk 1]
```diff
@@ -104,6 +126,10 @@ To define table data, the UI expects this:
 }
 ```
 
+### ${\color{yellow}IMPORTANT}$
+
+On Data values, `timestamp` column value must be in unix micro.
```
Comment (Ancairon) [inline src/collectors/plugins.d/functions-table.md:131]:
```suggestion
> **Important Note**
>
> On Data values, `timestamp` column value must be in unix micro.
```

**Review by Ancairon — COMMENTED** [review]
just a small tweak because yellow is not visible in light-mode

File: src/collectors/plugins.d/functions-table.md (hunk 1 above)
Comment (car12o) [reply src/collectors/plugins.d/functions-table.md]: changed, although I think it will remove the emphasis it should have

Comment (Ancairon) [reply src/collectors/plugins.d/functions-table.md]: If you want to have it as it was go for it, if this is not in Learn I won't gatekeep it, it's just that yellow was not looking good, and the preview style was odd to me

**Pull request merged**